mod/caching/spec/set/type_plus_right/wikirate_company/source_spec.rb

```diff
@@ -5,7 +5,7 @@
     samsung_sources = Card.fetch "Samsung+source"
     expect(samsung_sources.cached_count).to eq 0
     source = get_a_sample_source
-    company_list = source.fetch trait: :wikirate_company
+    company_list = source.fetch trait: :wikirate_company, new: {}
```

Comment:
I have `Company+*right` in the test db but the fetch is not returning me a virtual card. Do I miss sth?